builder/azure/arm/builder_acc_test.go

```diff
@@ -10,6 +10,8 @@ package arm
 // * ARM_CLIENT_SECRET
 // * ARM_SUBSCRIPTION_ID
 // * ARM_STORAGE_ACCOUNT
+// * AZURE_SSH_PASS - In our Linux SIG test we build two specialized images, this requires setting a shared password for both builds
+// * ARM_SSH_PRIVATE_KEY_LOCATION - the file location of a PEM encoded SSH private key,
```

Comment:
The location in the name is a little confusing to me. It is not clear to me if it is meant to be a file or directory on the remote or local filesystem.  It seems like this is just the path to a local SSH private key. Is that correct? 

If so what do you think about dropping Location from the name? 
```suggestion
// * ARM_SSH_PRIVATE_KEY_File - the file path of a local PEM encoded SSH private key to use in the test build,
```

Reply:
That's fine by me, I'd suggest KEY_FILE though, but I'll make that change